### PR TITLE
Tidy up for public release

### PR DESCRIPTION
**Pull request by Cleop**
Tackles issues from #43 broken into separate issues:
- #44 Creates a table of contents linking to each section
- #45 Adds dwyl colours for grid section
- #46  Thins and makes colors of input borders consistent
- #47 Ensures titles read 'Tachyons Bootstrap'
- #48 Adds github star count badge
- #51 Adds UA code script for Google Analytics

### CODE REVIEW

**Review by nelsonic — APPROVED** [review]
@Cleop Looks (_really_) Good! 🎉 
(_thanks for making time to do this_)

**Comment by iteles** [comment]
Odd. Tried to run this branch and localhost just hangs.

Will try again tomorrow after I try to shut down my computer overnight. Might  be a local thing.

**Review by iteles — COMMENTED** [review]
@cleop This is still hanging for me. Do you have 5 mins to review with me tomorrow please? 😊

File: index.html [hunk 1]
```diff
@@ -7,32 +7,54 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="style.css">
```
Comment (katbow) [inline index.html:10]: You'll need to serve this file as well. Right now it only serves the index & each of the components. Not set up to serve `.css` files yet 👍

**Review by katbow — CHANGES_REQUESTED** [review]
Looks good @Cleop! Just need to address the css file & should work a charm 👌 ✨

**Comment by Cleop** [comment]
Changes now made. Please now review @iteles and let me know if you encounter any more problems! @katbow please amend your review if you're happy.

File: component_combiner.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ var server = http.createServer(function (req, res) {
     homeHandler(req, res);
   } else if (components.indexOf(req.url) > -1) {
     componentHandler(req, res);
+  } else if (req.url === '/style.css') {
```
Comment (katbow) [inline component_combiner.js:20]: 👍

**Comment by katbow** [comment]
@Cleop Can a [test](https://github.com/dwyl/tachyons-bootstrap/blob/master/test/test.js) be added for the additions to `component_combiner.js`? That should keep it at 100% 🙂 💃

**Comment by Cleop** [comment]
Opps! Sorry for missing that. Here we go! @katbow

**Review by katbow — APPROVED** [review]
Great work @Cleop ! Thanks for updating 🎉

**Pull request merged**